### PR TITLE
chore(stack): add CustomResourcesURL to RuntimeConfig

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/deploy/cloudformation/stack/backend_svc_test.go [hunk 1]
```diff
@@ -217,41 +217,45 @@ Outputs:
...
+							StartPeriod: aws.Int64(0),
+							Timeout:     aws.Int64(10),
+						},
+						HostedZoneAliases: make(template.AliasesForHostedZone),
+						HTTPHealthCheck: template.HTTPHealthCheckOpts{
+							HealthCheckPath: manifest.DefaultHealthCheckPath,
+							GracePeriod:     aws.Int64(manifest.DefaultHealthCheckGracePeriod),
+						},
+						DeregistrationDelay: aws.Int64(60), // defaults to 60
+						CustomResources:     make(map[string]template.S3ObjectLocation),
```
Comment (efekarakus) [inline internal/pkg/deploy/cloudformation/stack/backend_svc_test.go:236]: The only thing added is this field. I've just changed the tests to use `DoAndReturn` and use `require.Equal` to compare objects as it works better than `gomock`.

`Gomock` would return an error when the field's location is different than expected

Comment (dannyrandall) [reply internal/pkg/deploy/cloudformation/stack/backend_svc_test.go:236]: great change this was so tedious before lol

**Review by dannyrandall — APPROVED** [review]
🥳

**Review by Lou1415926 — APPROVED** [review]
LGTM!

**Pull request merged**